editors/code/src/lsp.ts

```diff
@@ -30,11 +34,17 @@ export class Lsp {
 
 	private fileSettings: FileSettingsState;
 
+	public onSettingsNotificationEmitter: vscode.EventEmitter<void>;
+
 	constructor(context: vscode.ExtensionContext) {
 		this.channel = vscode.window.createOutputChannel("Air Language Server");
 		context.subscriptions.push(this.channel, registerLogger(this.channel));
+
 		this.stateQueue = new PQueue({ concurrency: 1 });
 		this.fileSettings = new FileSettingsState(context);
+
+		this.onSettingsNotificationEmitter = new vscode.EventEmitter<void>();
```

Comment:
It looks like `vscode.EventEmitter` has a `dispose()` method. Should it be in `context.subscriptions.push()`?